tests/reference/python-expr17-3b84714.stdout

```diff
@@ -0,0 +1,14 @@
+def if_check():
+    a: i32
+    a = 4
+    if (a) < (0):
+        print("negative value")
+    else:
+        if (a) > (0):
+            print("positive value")
+        else:
+            print("zero")
+
+
+
+
```

Comment:
Excess lines here should not be present. I think this can be handled in a separate PR.

Reply:
Yes, will create another PR for it.

Thanks!